src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.last {
+  margin-right: 50px;
+}
+
+.list {
+  padding: 0;
+}
+
+.header-1 {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.nav {
+  padding-left: 33.86px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.link {
+  letter-spacing: -0.01px ;
+  margin-left: -4.4px;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  padding-left: 20.5px;
+  margin-bottom: 5px;
+}
+
+.link:hover {
+  color: #00acdc ;
```

Comment:
Move repeatable colors to variable

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/2958/files#r1162644381